app/assets/stylesheets/_billables.scss

```diff
@@ -53,6 +53,26 @@
   }
 }
 
+.unbill {
+  display: inline-block;
+  background: none !important;
+  border: none;
+  border-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.